01LessonOne/index.html

```diff
@@ -1,4 +1,20 @@
 Your homework for today is take a pre-work review.
 
 <a href="https://docs.google.com/forms/d/1lOtAhvW-etoXPwtm64yRFqrpq5EsXjUlko2EiNFU3xw/viewform?c=0&w=1">Pre-work Review</a>
-<h1>helloworld</h1>
+<h1>Jason Carnline</h1>
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Jason Carnline</title>
+    </head>
+    <body> <hi>Car Dealership </h1>
+      <ol>
+<li>cars</li>
+<li>trucks</li>
+<li>boats</li>
+    </ol>
+    <herf= "http://google.com" img   </a>
```

Comment:
This line should be different. It appears you're trying to make an image a link, and we do that with <a href="example.com"> <img src="someimage.jpg"/> </a>. Notice the <img> tag is self closing, but the link has an opening and closing tag. Fix this, save, commit, and sync it receive full credit.